Makefile

```
@@ -0,0 +1,216 @@
+# Formatting/linting exclusions (from .pre-commit-config.yaml)
+ISORT_YAPF_EXCLUDES := --extend-exclude 'build/|sky/skylet/providers/ibm/|sky/schemas/generated/'
+IBM_FILES := sky/skylet/providers/ibm/
+
+##@ General
+
+help: ## Display this help message
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+##@ Setup
+
+install: ## Install dependencies using uv (creates venv if needed)
```

checkmake (0.2.2)

[warning] Target body for "install" exceeds allowed length of 5 (12).

(maxbodylength)